msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -209,6 +213,19 @@ To preview your app running in Office on the web:
 
     :::image type="content" source="images/office-web-more-apps.png" alt-text="Click on the 'More apps' option on the side bar of office.com to see your installed personal tabs":::
 
+### Office app for Android
+
+To view your app running in Office app for Android:
+
+1. Launch the Office app and sign in using your dev tenant account.
+1. Select the **Apps** icon. Your sideloaded app appears among installed apps.
+1. Select your app icon to launch your app in Office app for Android.
+
+> [!NOTE]
+> Before installing this app, verify you have followed [the steps To install the latest Office app beta build](prerequisites.md#mobile) and are part of the beta program.
```

Comment:
Please change to:
Before installing this app, perform [the steps to install the latest Office app beta build](https://review.learn.microsoft.com/en-us/microsoftteams/platform/m365-apps/prerequisites#mobile) and be a part of the beta program.